CHANGELOG.md

```diff
@@ -36,6 +36,7 @@
 - Faster startup by offloading glob matcher building to a worker thread #2868 (@cyqsimon)
 - Display which theme is the default one in basic output (no colors), see #2937 (@sblondon)
 - Display which theme is the default one in colored output, see #2838 (@sblondon)
+- Add support for Apple Silicon, see #2967 (@someposer)
```

Comment:
Support was already there (since you didn't have to do any code changes), so I think we should word this something like:

```suggestion
- Add aarch64-apple-darwin ("Apple Silicon") binary tarballs to releases, see #2967 (@someposer)
```